src/fees/hooks.ts

```diff
@@ -27,7 +27,9 @@ export function usePaidFees(fees: Fee[]) {
   const securityFeeAmount = fees.find((fee) => fee.type === TransactionFeeType.SecurityFee)
   const dekFeeAmount = fees.find((fee) => fee.type === TransactionFeeType.EncryptionFee)
   const feeCurrencyInfo = Object.entries(tokensByCurrency).find(
-    ([_, tokenInfo]) => tokenInfo?.address === securityFeeAmount?.amount.tokenAddress
+    ([_, tokenInfo]) =>
+      tokenInfo?.address === securityFeeAmount?.amount.tokenAddress ||
+      tokenInfo?.tokenId === securityFeeAmount?.amount.tokenId
```

Comment:
Adding this to be compatible with new versions that uses `tokenId`.
Another option would be to pass also the token address to the transaction confirmed event. Let me know if you prefer that solution.

Reply:
this seems okay, it is only used in the transfer item details screen which. might get a makeover soon anyway